DesignSystem/src/components/Atoms/LogoLoading.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const LogoLoading = () => (
+  <div className="logoLoading">
+      <div className="logoLoading-planet"></div>
+      <div className="logoLoading-orbit">
+        <div className="logoLoading-orbit-satelite"></div>
+      </div>
+  </div>
```

Comment:
I prefered to use the resources svg to do this but let's leave it like that

Reply:
Okey